docs/03-for-team-members/02-becoming-a-team-member.md

```diff
@@ -0,0 +1,42 @@
+# Team members
+
+Flathub, as a centralized repository of software apps, has a dedicated team of
+contributors necessary to keep it running. These contributors are in charge of:
+
+* Reviewing new applications
+* Reviewing permission changes for existing aplications
+* Requesting permission changes for existent apps that might not fit the
+  guidelines
+
+In consequence, they can be seen as the auditors and moderators of our
+community. As such, they play a big role in the decision-making process.
+They are also an important part of the Flathub and Flatpak trust model. The fact
+that applications' permissions are audited by the community is a warranty to
+consistent and safe permissions in the majority of the cases. Therefore, they
+also enjoy the recognition for their work!
+
+## Becoming a team member
```

Comment:
Should there be something along the lines of - `we can remove you from the team at any moment, without saying why we did it`

Just to make sure, that barth can remove/revoke people if things go south and it has been mentioned somewhere?

Reply:
If that's what you want, absolutely :) Though maybe instead there should be another section, like "revocation of access", "stepping down" or similar? How you want to do it, you tell me. Seems like that might make sense if you have Barth as BDFL?

Reply:
Just adding that line and adding another one like "if a team member wants to step down they can open an issue on flathub/flathub or can contact the admins via admin@flathub.org" is fine for now, imo.

Reply:
Added at the end of the section, as I could not fit it at the beginning without sounding aggressive